src/GitVersionCore.Tests/GitToolsTestingExtensions.cs

```diff
@@ -80,7 +81,14 @@ private static SemanticVersion ExecuteGitVersion(GitVersionContext context)
         /// </summary>
         public static void InitialiseRepo(this RemoteRepositoryFixture fixture)
```

Comment:
Perhaps `InitialiseRepo()` should be renamed to `InitializeRepo()` to be more consistent with the other American English parts of the codebase?

Reply:
Changed to `InitializeRepo`